book/routing.rst

```diff
@@ -1162,11 +1162,11 @@ each separated by a colon:
 
 For example, a ``_controller`` value of ``AppBundle:Blog:show`` means:
 
-=========  ==================  ==============
-Bundle     Controller Class    Method Name
-=========  ==================  ==============
-AppBundle  ``BlogController``  ``showAction``
-=========  ==================  ==============
+=============  ==================  ==============
+Bundle         Controller Class    Method Name
+=============  ==================  ==============
+``AppBundle``  ``BlogController``  ``showAction``
```

Comment:
I'm usually against using literals for bundle names (as it's a name and not code). However, in this case, the table is used to explain the `AppBundle:Blog:show` logical name. I think it makes sense to put it in a literal, as it's just the logical name broken into 3 seperate pieces.